lib/graphql/metrics/analyzer.rb

```diff
@@ -93,6 +96,17 @@ def result
 
       private
 
+      def prepare_arguments(node, definition)
```

Comment:
The issue it's fixing is with the `prepare` sub-methods, but this method is better named `get_arguments` or `arguments_for` or something.

Reply:
Is this method still going to be changed to `arguments_for`?